src/igvfd/schemas/multiplexed_sample.json

```diff
@@ -98,6 +99,22 @@
                 "igvf_utils": "TSTSM02881437, john-doe:tissue01, john-doe:primary_cell02"
             }
         },
+        "multiplexing_method": {
+            "title": "Multiplexing Method",
+            "description": "The method of multiplexing and/or demultiplexing.",
```

Comment:
can you explain a bit on what would be a 'multiplexing and demultiplexing' case?